ast/src/parsed/display.rs

```diff
@@ -142,17 +142,13 @@ impl Display for InstructionBody {
 }
 
 fn format_instruction_statement(stmt: &PilStatement) -> String {
```

Comment:
Can you create an issue to simplify the caller code and data structures so that we can have `stmt: &Expression` here?